src/libs/conduit/conduit_node.cpp

```diff
@@ -12468,6 +12477,7 @@ Node::to_base64_json(std::ostream &os,
     Node bb64_data;
     bb64_data.set(DataType::char8_str(enc_buff_size));
 
+#warning "I don't think to_base64_json should use the allocator, but it references mem"
```

Comment:
What happens if this is a GPU allocator?

Reply:
in this case, Node bb64_data is not a child of the current node, so it would use the default alloc and I think we are ok?

Reply:
ahh, i see below: we need to think about this -- call the memcpy method?

Reply:
I think this case is fine b/c, above the data is always compacted to a node that uses the default allocator.